host/sgx/linux/exception.c

```diff
@@ -39,7 +39,15 @@ static void _host_signal_handler(
     host_context.rip = (uint64_t)context->uc_mcontext.gregs[REG_RIP];
 
     // Call platform neutral handler.
-    uint64_t action = oe_host_handle_exception(&host_context);
+    uint64_t action;
+    if (!is_simulation(&host_context))
+    {
+        action = oe_host_handle_exception(&host_context);
+    }
+    else
+    {
+        action = oe_host_handle_exception_sim(context, sig_num);
```

Comment:
What about windows?

Reply:
The source file for Windows is another one host/sgx/windows/exception.c  , thanks!

Reply:
I didn't see any changes to the `host/sgx/windows/exception.c` in this PR. Is it intended?

Reply:
Sorry it is still in progress, now it is pushed.